Error: Cannot find module '@lando/memcached/builders/memcached.js'

Error Message: Error: Cannot find module '@lando/memcached/builders/memcached.js'

Looks like we're using the new "correct" patch for the memcached builder, but it's one of the very few plugins that hasn't updated to use that format.

I'm going make a quick release of the Symfony recipe using the old path (`@lando/memcached/services/memcached/builder.js`); [made an issue on the memcached plugin](https://github.com/lando/memcached/issues/30) to address the underlying issue, which will require some minor releases on all other recipes using memcached.

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -1,3 +1,7 @@
+## v1.1.0 - [February 19, 2024](https://github.com/lando/symfony/releases/tag/v1.1.0)
+
+* Fixed issue with @lando/memcached path being in an unexpected location. [#38](https://github.com/lando/symfony/issues/38)
+
 ## v1.0.0 - [December 13, 2023](https://github.com/lando/symfony/releases/tag/v1.0.0)
 
 * Dialed fully for `lando update`

--- a/builders/symfony-memcached.js
+++ b/builders/symfony-memcached.js
@@ -1,7 +1,7 @@
 'use strict';
 
 const _ = require('lodash');
-const LandoMemcached = require('@lando/memcached/builders/memcached.js');
+const LandoMemcached = require('@lando/memcached/services/memcached/builder.js');
 
 // Builder
 module.exports = {

